<launch>
    <node name="motor" pkg="control" type="motor" output="screen" >
        <param name="PortName" value="/dev/ttyUSB1" />
        <param name="control_topic" value="/cmd_vel"/>
        <param name="radar_result_topic" value="/robot/radar_result"/>
        <param name="chassis_status_topic" value="/robot/chassis_status"/>
        <param name="battery_voltage_topic" value="/robot/battery_voltage"/>
    </node>
    
</launch>
